frontend/src/components/filters.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  export let movieChecked = true
+  export let tvChecked = true
+  export let search
+
+  // If the cookie should break and set both to false
+  $: {
+    if (!movieChecked && !tvChecked) {
+      movieChecked = true
+      tvChecked = true
+    }
+  }
```

Comment:
I think this would look nice with an object and a more dynamic approach. This way we can just add more checks without having to do anything.

Will require a small adjustment in the markdown

```suggestion
  export let checked = {movie: true, tv: true}
  export let search

  // If the cookie should break and set both to false
  $: {
    if (Object.values(checked).every((v) => v === false)) {
      Object.keys(checked).forEach(v => checked[v] = true)
    }
  }
```

Reply:
I dont know if i agree with it being like that inside the component, the cookie Breaking is only from both values being false. Its very specific for this case where one checkbox is disabled if the other one is off. Dont Think other filters in the future would need to take part in this cookie checkup.

Also the object oriented way is already being used in the cookie. I Think its way easier and more clean to just use values in the component.

Reply:
Fair enough